modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -581,6 +581,16 @@ public Vector2(real_t x, real_t y)
             this.y = y;
         }
 
+        /// <summary>
+        /// Constructs a new <see cref="Vector2"/> from a scalar assigned to all components.
```

Comment:
I think we can use more common words here to be more beginner friendly.

```suggestion
        /// Constructs a new <see cref="Vector2"/> from a single value assigned to all components.
```

Reply:
Fixed